x-pack/plugins/canvas/server/templates/index.ts

```diff
@@ -13,20 +13,21 @@ import { light } from './theme_light';
 
 import { TEMPLATE_TYPE } from '../../common/lib/constants';
 
-export const templates = [pitch, status, summary, dark, light];
+export const templates = [status, summary, dark, light, pitch];
 
 export async function initializeTemplates(
-  client: Pick<SavedObjectsRepository, 'bulkCreate' | 'find'>
+  client: Pick<SavedObjectsRepository, 'bulkCreate' | 'create' | 'find'>
 ) {
   const existingTemplates = await client.find({ type: TEMPLATE_TYPE, perPage: 1 });
 
   if (existingTemplates.total === 0) {
-    const templateObjects = templates.map((template) => ({
-      id: template.id,
-      type: TEMPLATE_TYPE,
-      attributes: template,
-    }));
-
-    client.bulkCreate(templateObjects);
+    // Some devs were seeing timeouts that would cause an unhandled promise rejection
+    // likely because the pitch template is so huge.
+    // So, rather than doing a bulk create of templates, we're going to fire off individual
+    // creates and catch and throw-away any errors that happen.
+    // Once packages are ready, we should probably move that pitch that is so large to a package
+    for (const template of templates) {
+      client.create(TEMPLATE_TYPE, template, { id: template.id }).catch((err) => undefined);
```

Comment:
Is there any debug logging maybe we could do here that'd be appropriate? Just in case a dev does expect that template to get loaded